NameLayerMain/src/vg/civcraft/mc/namelayer/command/commands/ListSubGroups.java

```diff
@@ -0,0 +1,108 @@
+package vg.civcraft.mc.namelayer.command.commands;
+
+import java.util.List;
+import java.util.Set;
+import java.util.UUID;
+import java.util.logging.Level;
+
+import org.bukkit.ChatColor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+import com.google.common.collect.Sets;
+
+import vg.civcraft.mc.namelayer.GroupManager;
+import vg.civcraft.mc.namelayer.GroupManager.PlayerType;
+import vg.civcraft.mc.namelayer.NameAPI;
+import vg.civcraft.mc.namelayer.NameLayerPlugin;
+import vg.civcraft.mc.namelayer.command.PlayerCommandMiddle;
+import vg.civcraft.mc.namelayer.group.Group;
+import vg.civcraft.mc.namelayer.permission.GroupPermission;
+import vg.civcraft.mc.namelayer.permission.PermissionType;
+
+public class ListSubGroups extends PlayerCommandMiddle {
+
+	private static String format = "%s%s : (%s)\n";
+	
+	public ListSubGroups(String name) {
+		super(name);
+		setIdentifier("nllsg");
+		setDescription("List the nested group hierarchy.");
+		setUsage("/nllsg [group]");
+		setArguments(0,1);
+	}
+
+	@Override
+	public boolean execute(CommandSender sender, String[] args) {
+		if (!(sender instanceof Player)) {
+			sender.sendMessage(ChatColor.LIGHT_PURPLE + "No can do.");
+			return true;
+		}
+		
+		UUID uuid = NameAPI.getUUID(sender.getName());
+		
+		StringBuilder sb = new StringBuilder();
+		sb.append(ChatColor.GREEN);
+		sb.append("Group Hierarchy is as below:\n");
+		
+		try {
+			String groupname = args[0];
+			
+			if (!GroupManager.hasGroup(groupname)) {
+				sender.sendMessage(ChatColor.RED 
+						+ "Group does not exist.");
+				return true;
+			}
+			
+			if (!gm.hasAccess(groupname, uuid, PermissionType.GROUPSTATS)) {
+				sender.sendMessage(ChatColor.RED 
+						+ "You don't have permission to run that command.");
+				return true;
+			}
+			
+			Group group = GroupManager.getGroup(args[0]);
+			
+			sb.append(String.format(format, "", group.getName(), group.getMember(uuid)));
+			buildList(sb, uuid, group.getSubgroups(), "   ");
+		} catch (ArrayIndexOutOfBoundsException e) {
```

Comment:
You should just check how many arguments you have and go with a simple if else instead of taking the route over the exception here, this is a really bad habit.